File: Spring/les/src/main/java/com/jga/les/service/TelaService.java
package com.jga.les.service;

import com.jga.les.model.Tela;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Service;


@Service
public class TelaService extends GenericService<Tela> {

    public TelaService(JpaRepository<Tela, Long> objRepository) {
        super(objRepository, Tela.class);
    }

}

File: Flutter/les/lib/services/tela_service.dart

import 'package:dio/dio.dart';
import 'package:les/core/generic_service.dart';
import 'package:les/model/usuario/tela.dart';

class TelaService extends GenericService<Tela> {

  TelaService(Dio dio) : super(dio, 'tela', (data) => Tela.fromJson(data));

}